engine/classes/Elgg/Database/Seeds/Seed.php

```diff
@@ -500,15 +513,15 @@ public function getRandomUsername($base_name = 'user') {
 	/**
 	 * Set random metadata
 	 *
-	 * @param ElggEntity $entity   Entity
-	 * @param array      $fields   An array of profile fields in $name => $input_type format
-	 * @param array      $metadata Other metadata $name => $value pairs to set
+	 * @param \ElggEntity $entity       Entity
+	 * @param array       $group_fields An array of group profile fields in $name => $input_type format
+	 * @param array       $metadata     Other metadata $name => $value pairs to set
 	 *
-	 * @return ElggEntity
```

Comment:
There's no gain in returning these entities since we mutate them, and it makes tracking var types easier.